main.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable linebreak-style */
```

Comment:
- Please remove all disabled linters, remember that linters help us ensure that we write code that is clean and easy to maintain. Please fix this 🙏

Reply:
Suggested changes incorporated and disable statement removed screen shot attached for your kind reference 
![image](https://user-images.githubusercontent.com/117091923/220426669-cc079c6a-8a4a-479c-a2ce-91bf1041d97c.png)